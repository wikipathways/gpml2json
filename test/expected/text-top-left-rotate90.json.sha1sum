08689048cd7cad6598967ea02dced538d3bf5d47 ?-
