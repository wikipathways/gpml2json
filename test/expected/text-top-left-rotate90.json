{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["e933b"],"drawAs":"rect","gpmlElementName":"Pathway","height":212.1276595744681,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":554.2553191489361,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20171102","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/6017a024b6a2a3e31f42260d83ff0cbc676af277b67ddb76b73b754c6066ef33"},"entityMap":{"e933b":{"id":"e933b","textContent":"Label for\nthe rendering of\ntext","textAlign":"left","x":-0.5319148936169711,"y":67.55319148936174,"backgroundColor":"#ffffcc","fontSize":10,"height":100.95744680851064,"rotation":90,"drawAs":"rect","borderRadius":15,"width":156.27659574468078,"zIndex":16384,"color":"#000000","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"verticalAlign":"top","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"textRotation":-90,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"}}}
